spec/features/admin/manage_workshop_attendances_spec.rb

```diff
@@ -38,16 +38,5 @@
 
       expect(page).to have_content('2 are attending as students')
     end
-
-    xscenario 'can rsvp an invited student the the workshop' do
-      other_invitation = Fabricate(:workshop_invitation, workshop: workshop, attending: nil)
-
-      visit admin_workshop_path(workshop)
-      expect(page).to have_content('1 are attending as students')
-
-      select "#{other_invitation.member.full_name} (#{other_invitation.role})", from: 'workshop_invitations'
-
-      expect(page).to have_content('2 are attending as students')
-    end
```

Comment:
Hm. I wonder why this was commented out and if we should maybe try and fix this spec rather than removing it.